CONFIGURATION.md

```diff
@@ -0,0 +1,174 @@
+| `terms_file` | Path to the vocabulary JSON file containing allowed standardized term options for variable instances, relative to the configuration's directory | Yes | string | `"sexTerms.json"` |
+
+**Continuous Variables** (`data_type: "Continuous"`)
+| Field Name | Description | Required | Type | Example |
+| ---------- | -------------------------------------- | -------- | ----------------------- | -------------------------- |
+| `formats` | Array of allowed format options for column values | Yes | list of Format objects | See Format Configuration below |
+
+**Other Variables** (`data_type: null`)
+
+Variables with `data_type: null` are typically used for unique observation identifiers (e.g., Participant ID, Session ID) and multi-column measures (e.g., assessment tools). Multi-column measures require a `terms_file` to define the available standardized terms.
+
+| Field Name   | Description                                                                                                                                     | Required | Type   | Example                  |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------ | ------------------------ |
+| `terms_file` | Path to the vocabulary JSON file containing allowed standardized term options for variable instances, relative to the configuration's directory | Yes\*    | string | `"assessmentTerms.json"` |
+
+\*Required only for multi-column measures (`is_multi_column_measure: true`).
+
+### Format Configuration
+
+When specifying `formats` for continuous variables, each format object must have the following fields:
+
+| Field Name | Description                            | Required | Type            | Default Value | Example            |
+| ---------- | -------------------------------------- | -------- | --------------- | ------------- | ------------------ |
+| `id`       | Unique identifier for the format       | Yes      | string          | -             | `"FromFloat"`      |
+| `name`     | Human-readable name for the format     | Yes      | string          | -             | `"float"`          |
+| `examples` | Array of example values in this format | No       | list of strings | `[]`          | `["24.2", "32.3"]` |
+
+For an example `formats` field, see the Example Configuration below.
+
+### Example Configuration
+
+```json
+[
+  {
+    "vocabulary_name": "Neurobagel Phenotypic Variables",
+    "namespace_prefix": "nb",
+    "namespace_url": "http://neurobagel.org/vocab/",
+    "version": "1.0.0",
+    "standardized_variables": [
+      {
+        "name": "Participant ID",
+        "id": "ParticipantID",
+        "data_type": null,
+        "terms_file": null,
+        "formats": null,
+        "required": true,
+        "description": "Describe how to annotate this standardized variable.",
```

Comment:
🍒 
instead of this filler text, could we use the example descriptions from the Neurobagel config?